### DISCUSSION TITLE
Advanced Install Mac Address

### DISCUSSION THREAD

**Discussion by ianalexis** [tteck/Proxmox#382] [Ideas]
To facilitate the CTs creation i think we could add a mac address option in the Advanced mode in case already have one assigned with the ip i want.
This could be useful on the WireGuard install for example.

**Comment by btehan** [comment]
Another use case for this is when you want to re-do an LXC from scratch and you already have a static reservation in your router.

**Reply by ianalexis** [reply]
IPs, Port Forwarding, Reverse Proxy, etc, etc, etc...

**Comment by tteck** [comment]
@ianalexis @btehan For an early look at the v4 script
```yaml
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/ubuntu-v4.sh)"
```
If you notice that something is missing, please let me know

**Comment by btehan** [comment]
Looks great to me.  I also like the whiptail menus :-)
Thanks!
